testtools/tests/test_deferredruntest.py

```diff
@@ -620,11 +622,17 @@ def test_something(self):
         result = self.make_result()
         runner.run(result)
         self.assertThat(
-            result._events,
+            [event[:2] for event in result._events],
```

Comment:
I guess doing this will hide any point in the future where any of these events grow to be more than 2 items long. Not sure if that matters or not.

An alternative would be to get tricky with MatchesListwise I guess.